Physics/CdrPlasma/PlasmaModels/CdrPlasmaJSON/CD_CdrPlasmaJSON.H

```diff
@@ -647,7 +649,46 @@ namespace Physics {
       /*!
 	@brief List of dielectric reactions
       */
-      std::vector<CdrPlasmaSurfaceReactionJSON> m_dielectricReactions;      
+      std::vector<CdrPlasmaSurfaceReactionJSON> m_dielectricReactions;
+
+       // ===========================================
+      // DOMAIN SURFACE REACTIONS BEGIN HERE
+      // ===========================================
+
+      /*!
+	@brief Lookup method for the domain reaction ratesThe pair is made up of an int representing direction (0=x, 1=y, 2=z) and a Side::LoHiSide
+	representing side (Side::Lo, Side::Hi)
+      */
+      std::map<std::pair<int, Side::LoHiSide>, std::map<int, LookupMethod>> m_domainReactionLookup;
```

Comment:
To be kind to older compilers, let '>>' be separate by a space. I.e. '> >'. Older compilers will complain.